app/helpers/change_log_helper.rb

```diff
@@ -0,0 +1,18 @@
+module ChangeLogHelper
+
+  def updated_by_user(revision)
+    if revision.tracked_changes[:updated_by_id].present?
+      user_guid_to_email(revision.tracked_changes[:updated_by_id][:to])
+    else
+      "N/A"
```

Comment:
Will it ever actually hit this condition, since it's defaulting to `system` in `user_guid_to_email`?